mesonbuild/interpreter/interpreter.py

```diff
@@ -70,6 +70,7 @@
 import typing as T
 import textwrap
 import importlib
+import argparse
```

Comment:
Since this only affects typing stuff this should go into the `if T.TYPE_CHECKING:` block. You will have to use `'argparse.Namespace'` then but it reduces the number of actual imports.

However, this is only a very minor style nitpick.

Reply:
done.